runtime/databricks/automl_runtime/forecast/prophet/model.py

```diff
@@ -239,7 +239,10 @@ def predict_timeseries(self, horizon: int = None, include_history: bool = True)
             identity_column_names=self._id_cols
         )
         future_df["ts_id"] = future_df[self._id_cols].apply(tuple, axis=1)
-        return future_df.groupby(self._id_cols).apply(lambda df: self._predict_impl(df, horizon, include_history)).reset_index()
+        print(f"predict_timeseries::future_df:\n {future_df}")
+        return future_df.groupby(self._id_cols).apply(
+                lambda df: self._predict_impl(df, horizon, include_history)
+            ).reset_index(drop=True)
```

Comment:
Here I'm adding `drop=True` to resolve some ValueError.
I think adding `drop=True` is ok, as we have already manually added back the id cols in `_predict_impl`. But I do want to double check this with you. @lu-wang-dl